base_openstack_ubuntu-16.04.json

```diff
@@ -0,0 +1,54 @@
+{
+  "builders": [{
+    "name": "openstack_ubuntu_16.04_base",
+    "type": "openstack",
+    "ssh_username": "ubuntu",
+    "image_name": "ubuntu_base_{{user `build_timestamp`}}",
+    "source_image": "3eafe7dd-d90d-4d8e-ab70-e0c757a11bf7",
+    "use_floating_ip": true,
+    "floating_ip_pool": "ext-net",
+    "security_groups": ["default", "remote SSH", "remote HTTP", "remote https"],
+    "networks": ["5b8760e4-06ed-4f73-8c8d-bb1543b0bc74"],
+    "flavor": "m1.small",
+    "ssh_keypair_name": "jmatt_lsst",
+    "ssh_private_key_file": "/Users/jmatt/.ssh/id_rsa_lsst",
+    "ssh_timeout": "30m"
+  }],
+  "provisioners": [
+    {
+      "type": "shell",
+      "execute_command": "echo 'ubuntu' | {{.Vars}} sudo -S -E sh -eux '{{.Path}}'",
+      "inline": [
+	"ping -w 600 -c 2 -i 10 141.142.2.2",
+	"ping -w 600 -c 2 -i 10 8.8.8.8",
+	"ping -w 600 -c 2 -i 10 archive.ubuntu.com",
+	"ping -w 600 -c 2 -i 10 security.ubuntu.com",
```

Comment:
To work around nonsense from Openstack Neutron. The server will be able to ssh and connect before it'll be able to resolve internal and external routes. This is a workaround that fails significantly less.